Modules/_collectionsmodule.c

```diff
@@ -951,7 +952,7 @@ deque_reverse(dequeobject *deque, PyObject *unused)
 }
 
 PyDoc_STRVAR(reverse_doc,
-"D.reverse() -- reverse *IN PLACE*");
+"Reverse the elements of the deque *IN PLACE*.");
```

Comment:
Any reason to not do

```suggestion
"reverse()\n\n"
"Reverse the elements of the deque *IN PLACE*.");
```

Reply:
Thanks for the review!
I guess I omitted it initially because it does not add information, but for consistency reasons it totally makes sense.